site/docs/tasks/multicluster.md

```diff
@@ -106,3 +106,80 @@ For the remote cluster the coordinator will be disabled:
 coordinator:
     enabled: false
 ```
+
+
+## Configure Vault for multi-cluster deployment
+
+The Fybrik uses [HashiCorp Vault](https://www.vaultproject.io/) to provide running Fybrik modules in the clusters with the dataset credentials when accessing data. This is done using [Vault plugin system](https://www.vaultproject.io/docs/internals/plugins) as described in [here](../concepts/vault_plugins.md).
+
+
+This section describe the steps to enable the modules to authenticate to Vault in order for them to retrieve the dataset credentials.
+Some of the steps described below are not specific to the Fybrik project but rather are Vault specific and can be found in Vault related online tutorials.
+
+Module authentication is done by configuring Vault to use [Kubernetes auth method](https://www.vaultproject.io/docs/auth/kubernetes) in each cluster. Using this method the modules can authenticate to Vault by providing their service account token. Behind the scenes Vault authenticates the token by submitting TokenReview request to the API server of the kubernetes cluster where the module is running.
+
+### Prerequisites unless Fybrik modules are running on the same cluster as the Vault instance:
+
+1. The running Vault instance should have connectivity to the cluster API server for each cluster running Fybrik modules.
+2. The running Vault instance should have an Ingress resource to enable Fybrik modules getting credentials.
+
+### Enabling Kubernetes auth for each cluster with running Fybrik modules:
+
+1. Create a token reviewer service account called vault-auth in the fybrik-system namespace and give it permissions to create tokenreviews.authentication.k8s.io at the cluster scope:
```

Comment:
Create a token reviewer service account called `vault-auth` in the `fybrik-system` namespace and give it permissions to create `tokenreviews.authentication.k8s.io` at the cluster scope: